editors/vscode/README.md

```diff
@@ -5,15 +5,26 @@ This extension provides support for the [Lambdapi](https://github.com/Deducteam/
 Goals are visualised in a panel on the right side of the editor.
 You can navigate in proof with the following key-bindings:
 
-- ``Ctrl+Right``: go one step forward
-- ``Ctrl+Left``: go one step backward
-- ``Ctrl+Up``: go to the previous proof (or the beginning)
-- ``Ctrl+Down``: go to the next proof (or the end)
+***Linux and Windows***
+- ``Ctrl+Right`` (``Ctrl+fn+Right`` in `Mac OS X`): go one step forward
+- ``Ctrl+Left`` (``Ctrl+fn+Left`` in `Mac OS X`): go one step backward
+- ``Ctrl+Up``: go to the previous proof <sup>*</sup> (or the beginning)
+- ``Ctrl+Down``: go to the next proof <sup>*</sup> (or the end)
+- ``Ctrl+Enter``: go to the position of the cursor
+- ``Ctrl+Alt+c``: toggle cursor mode (proof highlight follows the cursor or not)
+- ``Ctrl+Alt+w``: toggle follow mode (proof highligsht is always centered in the window when keybindings are pressed)
+- ``Shift+Alt+w``: center proof highlight in the current window
+
+***Mac OS X***
+- ``Ctrl+fn+Right``: go one step forward
+- ``Ctrl+fn+Left``: go one step backward
 - ``Ctrl+Enter``: go to the position of the cursor
 - ``Ctrl+Alt+c``: toggle cursor mode (proof highlight follows the cursor or not)
 - ``Ctrl+Alt+w``: toggle follow mode (proof highlight is always centered in the window when keybindings are pressed)
 - ``Shift+Alt+w``: center proof highlight in the current window
 
+For `go to the previous proof` and `go to the next proof`, Key bindings need to be changed in Code->Preferences->keyboard shortcuts (also reachable with Command+K Command+S) because default ones are used by Mac OS X
```

Comment:
Why not change the default ones so that users won't have to do that.
My previous remark was just about saying how to change any binding if someone wants.

Reply:
The problem is that changing the shortcut will impact the user under other OSs (Linux and Windows) I am not sure this is desirable and how such users will appreciate this change. We could also change the shortcuts, advertise this change and document how to restore the old shortcuts back for Linux and Windows users. What do you think?

Reply:
Is it possible to have different shortcuts depending on the OS?

Reply:
Btw, a last typo: ", Key" -> ", key".

Reply:
> Is it possible to have different shortcuts depending on the OS?

I believed it was not possible but it seems it is. I have to check though.

Reply:
I didn't know of `isWindow`, `isLinux` and `isMac` context keys. I used them to define different shortcuts for Mac OS